File: Lógica/Lista3/ExPySalaLista/ex11.py
n = int(input("Digite um número: "))
maior = n 
menor = n
for i in range(1,4):
    n = int(input("Digite um número: "))
    if n > maior:
        maior = n  
    elif n < menor:
        menor = n
        
print(f"Menor número: {menor} \n Maior número: {maior}")

File: Lógica/Lista3/ExSalaC/ex11.c
#include <stdio.h>

int main() {
	int i,n,maior,menor,hold;
    
    
    for(i = 1; i <= 3;++i){
        printf("Digite o número %i: ",i);
        scanf("%i",&n);
        if (i == 1){
            maior = n;
            menor = n;
            continue;
        }
        
        if (n > maior){
            maior = n;
        }
        if (n < menor){
            menor = n;
        }
            
    }
    
    printf("Menor: %i \n",menor);
    printf("Maior: %i",maior);
 
	return 0;
}